subsys/bluetooth/mesh/Kconfig

```diff
@@ -141,23 +150,23 @@ config BT_MESH_NODE_ID_TIMEOUT
 	  be the appropriate value as well, so just leaving this as the
 	  default is the safest option.
 
+config BT_MESH_PROXY_USE_DEVICE_NAME
+	bool "Include Bluetooth device name in scan response"
+	depends on BT_MESH_GATT_PROXY
+	help
+	  This option includes GAP device name in scan response when
+	  the GATT Proxy feature is enabled.
+
 config BT_MESH_PROXY_FILTER_SIZE
 	int "Maximum number of filter entries per Proxy Client"
 	default 3 if BT_MESH_GATT_PROXY
 	default 1
 	range 1 32767
-	depends on BT_MESH_PROXY
+	depends on BT_MESH_GATT_SERVER
```

Comment:
Also should be depends on BT_MESH_GATT_PROXY by ideally.
But now being coupled by pb-gatt is really annoying, but I will solve this problem completely in the next pr.